catalog/core/catalog-core-solr/src/main/java/ddf/catalog/source/solr/SolrFilterDelegate.java

```diff
@@ -666,6 +667,11 @@ private Geometry getGeometry(String wkt) {
     private String sanitizeGeometry(String wkt) {
         try {
             Shape wktShape = WKT_READER.read(wkt);
+            //All polygons will be an instance of JtsGeometry. If it is not a polygon we don't need
```

Comment:
I am also wondering if the reverse is necessarily true? i.e. `All polygons are JtsGeometry` does not imply `All JtsGeometry are polygons`.

Is there a source for this fix, maybe on the mailing lists or something? Agree with @lessarderic that we should be as clear as possible here.